### PR TITLE
feature: Add f5apm provider

### PR DESCRIPTION
**Pull request by six-arm**
Support for F5's Access Policy Manager as an additional identity provider. 
* Automatic detection of MFA
* Automatic detection of MFA options (push, token)

### CODE REVIEW

**Comment by six-arm** [comment]
Here's an example of a passing build for the branch.  https://travis-ci.com/six-arm/saml2aws/builds/103085564.  Please let me know what else is needed for considering this PR.

File: pkg/cfg/cfg.go [hunk 1]
```diff
@@ -62,18 +65,23 @@ func (ia IDPAccount) String() string {
...
   RoleARN: %s
-}`, appID, ia.URL, ia.Username, ia.Provider, ia.MFA, ia.SkipVerify, ia.AmazonWebservicesURN, ia.SessionDuration, ia.Profile, ia.RoleARN)
+}`, appID, policyID, ia.URL, ia.Username, ia.Provider, ia.MFA, ia.SkipVerify, ia.AmazonWebservicesURN, ia.SessionDuration, ia.Profile, ia.RoleARN)
 }
 
 // Validate validate the required / expected fields are set
 func (ia *IDPAccount) Validate() error {
-	if ia.Provider == "OneLogin" {
+	switch ia.Provider {
+	case "OneLogin":
```
Comment (wolfeidau) [inline pkg/cfg/cfg.go]: Is there a reason your updating the onelogin related code?

Comment (wolfeidau) [reply pkg/cfg/cfg.go]: Ideally you would keep your change just to the scope of adding your provider, and then any other fixes would be seperate.

Comment (six-arm) [reply pkg/cfg/cfg.go]: I missed that I changed the message in the onelogin block.  I'll remove this and resubmit.  Thanks.

**Comment by six-arm** [comment]
@wolfeidau, please let me know if anything else is needed to resolve this PR, thanks.

**Pull request merged**